src/core/grid_based_algorithms/lb_particle_coupling.hpp

```diff
@@ -10,6 +12,36 @@ void lb_lbcoupling_calc_particle_lattice_ia(bool couple_virtual);
 
 uint64_t lb_lbcoupling_get_rng_state();
 void lb_lbcoupling_set_rng_state(uint64_t counter);
+void lb_lbcoupling_set_friction(double friction);
+double lb_lbcoupling_get_friction();
 void mpi_set_lb_coupling_counter_slave(int high, int low);
+void mpi_bast_lb_particle_coupling(int, int);
+
+struct LB_Particle_Coupling {
+  Utils::Counter<uint64_t> rng_counter_coupling;
+  /*
+   * @brief Salt for the RNGs
+   *
+   * This is to avoid correlations between the
+   * noise on the particle coupling and the fluid
+   * thermalization.
+   */
+  enum class RNGSalt { PARTICLES = 2 };
```

Comment:
This is explicitly _not_ an property of the coupling. Please make this one enum again and put it into a common header. This is so that the compiler generates the numbers and make sure they don't collide.